### ISSUE TITLE
执行官方推理代码，弹出pad token id 和attention_mask需要被添加的警告

### ISSUE THREAD

**Issue by ruio248** [QwenLM/Qwen1.5#133]
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation.

**Comment by JustinLin610** [comment]
Just ignore it.

**Comment by jklj077** [comment]
closed in favor of https://github.com/QwenLM/Qwen1.5/issues/26
